web/app_dev.php

```diff
@@ -18,7 +18,10 @@
     exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
 }
 
-$loader = require_once __DIR__.'/../app/bootstrap.php.cache';
+/**
+ * @var Composer\Autoload\ClassLoader $loader
+ */
+$loader = require __DIR__.'/../app/autoload.php';
```

Comment:
But, ezpublish, for example, already does this (loading a different file based on the environment): https://github.com/ezsystems/ezpublish-community/blob/master/web/index.php#L24. If they haven't had issues, I would think we also would be ok.